### ISSUE TITLE
Broken again with HA Core update 2023.1.7

### ISSUE THREAD

**Issue by Salsalove** [Ligio/hacc-ozmo#43]
2023-01-24 11:28:22.431 ERROR (mqtt_schedule_thread) [ozmo] call to iotdevmanager failed with {'ret': 'fail', 'errno': 4200}

**Comment by Salsalove** [comment]
Solved! 
I don't why, but in the configuration.yaml "contintent: na" doesn't work anymore, you have to use "contintent: ww" 😉

**Comment by noiob** [comment]
same for EU, changing the config to "ww" fixed it

**Comment by Salsalove** [comment]
After last HA update (2023.3) this custom component is broken again!

**Comment by Arn0uDz** [comment]
For me this integration is still working, even on 2023.3.1 , the only problem i'm having over all versions of HA is that this integration stops working after about a week and gives me the above error message. I have to reboot HA in order for it to work again, rinse and repeat.

**Comment by Salsalove** [comment]
Yes...now it's working for me too. I had to reinstall the component, but i don't know why 🤷🏼

I reboot HA every night with an automation, so i don't have your issue.
